apps/yapms/src/lib/components/modals/importmodal/ImportModal.svelte

```diff
@@ -50,122 +74,100 @@
 		});
 	}
 
+	function loadCustomMap() {
+		if (files === undefined || files === null) {
+			return;
+		}
+
+		if (fileType === '' || fileType === 'invalid') {
+			return;
+		}
+
+		if (fileType === 'application/geo+json') {
+			load(importFromGeoJson, files);
+		} else if (fileType === 'shp') {
+			load(importFromShapefiles, files);
+		} else if (fileType === 'image/svg+xml') {
+			load(loadSVG, files);
+		}
+	}
+
 	function close() {
 		$ImportModalStore.open = false;
-		if ($page.url.pathname === '/app/imported' && $ImportedSVGStore.loaded === false) {
+		if (page.url.pathname === '/app/imported' && $ImportedSVGStore.loaded === false) {
 			goto('/');
 		}
 	}
 </script>
 
-<ModalBase title="Import Map" store={ImportModalStore} onClose={close}>
+<ModalBase title="Custom Map" store={ImportModalStore} onClose={close}>
 	<div slot="content">
 		<div class="flex flex-col gap-y-2">
-			<div class="alert alert-error justify-start" class:hidden={!loadError}>
-				<span class="flex gap-x-2"
-					><ExclamationCircle class="w-6 h-6" />There was an error loading your map, please try
-					again.</span
-				>
-			</div>
-			<div class="alert alert-info justify-start" class:hidden={!loading}>
-				<span class="flex gap-x-2"><ArrowUpTray class="w-6 h-6" />Loading Map...</span>
-			</div>
 			<div class="flex flex-col gap-y-2">
-				<fieldset class="fieldset w-full flex gap-x-2">
-					<legend class="fieldset-legend flex gap-x-1">
-						<p class="text-lg">Open From GeoJson</p>
-						<div class="tooltip tooltip-bottom" data-tip={multipleFilesTooltip}>
-							<DocumentDuplicate class="w-6" />
+				<fieldset class="fieldset w-full">
+					<legend class="fieldset-legend">Map Files</legend>
+					<div class="flex gap-x-2">
+						<input
+							multiple
+							type="file"
+							accept=".geojson, .json, .svg, .shp"
+							class="file-input w-full"
+							class:file-input-error={loadError || fileTypeIsInvalid}
+							disabled={loading}
+							bind:files
+						/>
+					</div>
+					{#if loadError === true}
```

Comment:
loadError is should be cleared on a successful load (an oversight of mine, actually!)